More accurate phrasing of emblems under IHL (#31)
rahelFain/combined-diem-uses-reqs: draft-ietf-diem-requirements.md

Reviewer: I think that the term Digital Protective Emblem for four use cases must not be added to the draft.  See a further comment in this review. I also think that the new reference to the Unesco Blue Shield document (which I read through) is misleading, because there are two differing statements  in there, on page 16. One list uses with digital assets, and  aligns with the ICRC, but further down the page is a second statement: "The digital Blue Shield emblem will also facilitate visualization on the UNESCO Sites Navigator of  cultural  properties".   This appears to allow for a digital emblem for a physical location because Sites Navigator is a GIS database. So the UNESCO reference should not be included or used as is.  These two concerns lead me to rejecting this PR.

Reviewer 2: The references here were used in the mailing list as a means of arguing that the ICRC's definition of digital emblems on the mailing list. In light of this and the potential issues with the ITU I also feel I need to reject these changes.

IAEA administers several treaties, especially related to the controlled shipment of atomic fuels and wastes across borders.
Similar use case as OPCW.

> ## Protective Emblems under International Humanitarian Law

Reviewer 3: This would be way clearer

```suggestion
## Protective Emblems under The Geneva Conventions and its Additional Protocols
```

Author: Heya, agree and disagree at the same time. Agree: Listing the concrete laws is better and more accurate. Disagree: Blue shield is defined in the Hague Convention, so that's missing. Adding a third law/document/whatever to the text really becomes silly though.

As far as I'm aware, these emblems *are* the protective emblems under IHL, and that is how they are referred to, so I would leave it as is. The body of the section also is very accurate and clear about what we're referring to.

Reviewer 3: Without going into the gory details, International Humanitarian Law provides various levels of protection for a lot of things (ex: water treatment plants, schools), especially things that disproportionately impact civilians, that do not have explicit physical emblems defined. These rules are subtle. However if someone defined an emblem for schools, that Emblem would be intended to communicate this (limited) Protection under Internal Humanitarian Law. That is awfully close to the title. What is unique about the 4 emblems is that they are specifically called out as emblems in the Geneva Conventions + APs, and that 3 of them are defined there.

### Background

> The Geneva Conventions and their Additional Protocols constitute the core of IHL.

Reviewer 3: ```suggestion
The Geneva Conventions and their Additional Protocols constitute the core of International Humanitarian Law (IHL).
```

> Some assets enjoy certain specific protections under IHL, including that they must not be attacked, and IHL codifies four types of protective emblems for armed conflict, which inform other parties that marked assets benefit from one or several of these specific or special protections.

Reviewer 3: This clarifies the distinction between other signs like a white flag and the UN flag and logo, and the four emblems mentioned in the GP+APs.

```suggestion
Some assets enjoy certain specific protections under IHL, including that they must not be attacked. In addition to recognizing other signs, IHL codifies four types of protective emblems for armed conflict, which inform other parties that marked assets benefit from one or several of these specific or special protections.
```

In other words, protective emblems under IHL signal the applicability of a specific or special protection under IHL.
Namely, these emblems are:

- The emblems of the Red Cross, Red Crescent, and Red Crystal, defined in the Geneva Conventions and Additional Protocol III of the Geneva Conventions {{RCRCRES}}
- The Blue Shield emblem (a shield, pointed below, per saltire blue and white), defined in the 1954 Hague Convention {{UNESCORES}}
- The international distinctive sign of civil defence, an equilateral blue triangle on an orange ground, defined in Additional Protocol I of the Geneva Conventions

> - The dangerous forces special sign, consisting of a group of three bright orange circles placed on the same axis, defined in Additional Protocol I of the Geneva Conventions

Reviewer 3: - I like spelling out the locations of the definition of the emblems (ex: AP I, GP and AP III), but I think the description of the physical emblem is distracting.
- the references should be to the original convention/treaty/law defining an emblem (GP, AP I, AP III, Hague Convention)

However, these emblems can currently only be used to mark physical assets, and there is no way to mark digital, network-connected infrastructure that enjoys the same protections.

> A digital emblem using the DIEM architecture could address this gap, and we call such emblems Digital Protective Emblems under IHL.

Reviewer: Adding a new term (Digital Protective Emblems is overkill - each use case is able to specify the specific requirement when they implement DEs, and how they will meet them,  so having two distinct types, the DE and the DPE, isn't necessary.  The four use cases  that received this new term are already well described here, and grouped already, so  the document giving them their own term is unnecessary as well as potentially confusing.

> A digital emblem using the DIEM architecture could address this gap, and we call such emblems Digital Protective Emblems under IHL.

Reviewer 3: Defining a new capitalized term in a non-normative section of the document is simply not appropriate. Why not just say in line 325: 
"In context of the emblems above described in the GP..."?

Similar use case as OPCW.

## Protective Emblems under International Humanitarian Law

### Background

The Geneva Conventions and their Additional Protocols constitute the core of International Humanitarian Law (IHL).
Some assets enjoy certain specific protections under IHL, including that they must not be attacked. In addition to recognizing other signs, IHL codifies four types of protective emblems for armed conflict, which inform other parties that marked assets benefit from one or several of these specific or special protections.
In other words, protective emblems under IHL signal the applicability of a specific or special protection under IHL.
Namely, these emblems are:

- The emblems of the Red Cross, Red Crescent, and Red Crystal, defined in the Geneva Conventions and Additional Protocol III of the Geneva Conventions {{GCI1949}} {{APIII2005}} {{RCRCRES}}
- The Blue Shield emblem, defined in the 1954 Hague Convention {{HAGUE1954}} {{UNESCORES}}
- The international distinctive sign of civil defence, defined in Additional Protocol I of the Geneva Conventions {{API1977}}

> - The dangerous forces special sign, defined in Additional Protocol I of the Geneva Conventions {{API1977}}

Reviewer 3: ```suggestion
- The emblems of the Red Cross, Red Crescent, and Red Crystal, defined in the Geneva Conventions and Additional Protocol III of the Geneva Conventions {{GCI1949}} {{APIII2005}}
- The Blue Shield emblem, defined in the 1954 Hague Convention {{HAGUE1954}}
- The international distinctive sign of civil defence, defined in Additional Protocol I of the Geneva Conventions {{API1977}}
- The dangerous forces special sign, defined in Additional Protocol I of the Geneva Conventions {{API1977}}
```

Author: Why do you want to remove these references?
